### PR TITLE
fix recall message

### PR DESCRIPTION
**Pull request by wxul**
Related to this [PR](https://github.com/Chatie/wechaty-puppet-puppeteer/pull/81), some code has been changed

### CODE REVIEW

**Comment by huan** [comment]
Why did you close the original PR #81?

**Comment by wxul** [comment]
Because of I made some mistakes when I updated this PR. So I close that PR and push a new one. I think we can discuss in the Issues?

**Comment by huan** [comment]
It's not good practice if you create a new PR in order to replace an old PR.

The better practice should be continuing to modify the original PR until it had been merged.

File: src/pure-function-helpers/web-message-type.ts [hunk 1]
```diff
@@ -52,7 +52,9 @@ export function webMessageType (
      */
     case WebMessageType.SYS:
       return MessageType.Text
-
+    // add recall type
```
Comment (windmemory) [inline src/pure-function-helpers/web-message-type.ts]: I think we don't need the comment here, since the type is clearly stated it's purpose.

**Comment by huan** [comment]
@windmemory Thanks for the approvement.

@wxul Thank you very much for the contribution, you did a great job!

**Pull request merged**